core/src/text/fontContext.cpp

```diff
@@ -185,6 +185,11 @@ bool FontContext::layoutText(TextStyle::Parameters& _params, const std::string&
                      (metrics.aabb.y + height * 0.5) * TextVertex::position_scale);
 
     auto it = _quads.begin() + quadsStart;
+    if (it == _quads.end()) {
```

Comment:
What will happen currently if a label is missing a glyph, will it show a default quad or an empty space?

If so, then yes it makes sense to drop the label if its missing any glyph required by it to be rendered completely.